opensoundscape/datasets.py

```diff
@@ -219,6 +219,10 @@ class SingleTargetAudioDataset(torch.utils.data.Dataset):
         overlay_weight: The weight given to the overlaid image during augmentation.
             When 'random', will randomly select a different weight between 0.2 and 0.5 for each overlay
             When not 'random', should be a float between 0 and 1 [default: 'random']
+        overlay_class: The label of the class that overlays should be drawn from.
```

Comment:
the user should have to "get this right" by selecting the absence class label for binary model. As is, they could run with the default values and repeat our current bug.

Reply:
Added error-checking to force user to "get it right" by either selecting absence class label or specifying the multi-class behavior with `overlay_class = 'different'`